documentation/modules/appendix_crds.adoc

```diff
@@ -886,6 +886,8 @@ include::../api/io.strimzi.api.kafka.model.template.PodTemplate.adoc[leveloffset
 
 
 |https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#HostAlias-v1-core[HostAlias] array
```

Comment:
Please reword to "a Pod's".

Indicates whether information about services should be injected into a Pod's environment variables.

Does it make sense to add the example you used in the PR description?